papers/__main__.py

```diff
@@ -31,6 +31,9 @@ def check_legacy_config(configfile):
 
 
 def get_biblio(config):
+    """
+    Given a config, get the Biblio object associated with this run.  If there's no Biblio object, will make a new one, silently.
```

Comment:
This function initializes a Biblio object based on the bibtex file specified as command line argument or in config file. It no bibtex file is specified, it throws an error.

Reply:
Set in https://github.com/perrette/papers/pull/55/commits/9d195646e9e239de8c8a59f7e118298eb1a29797